model/api/src/dataRecordPathFns.ts

```diff
@@ -1,7 +1,7 @@
 import {
-  DataRecordPath,
-  DataRecordPathElement,
-  Property,
+  type DataRecordPath,
+  type DataRecordPathElement,
+  type Property,
   propertyDescriptors,
 } from '@cozemble/model-core'
 
```

Comment:
have you tested this mate?  I have a memory of svelte:check not liking this.  Sorry not this case, the case above.  If all the imported things are `type` I think `svelte:check` wants `import type {....}`

Reply:
Yes, no problems from build, IDE or linting. `propertyDescriptors` is not a type